src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/DisplayFormatAttribute.cs

```diff
@@ -11,6 +13,9 @@ namespace System.ComponentModel.DataAnnotations
     [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field, AllowMultiple = false)]
     public class DisplayFormatAttribute : Attribute
     {
+        private readonly LocalizableString _nullDisplayText = new LocalizableString("NullDisplayText");
+        private Type _nullDisplayTextResourceType;
```

Comment:
FYI (mostly) this field duplicates `_nullDisplayText.ResourceType`. Not sure why similar `_resourceType` was kept separate in `[Display]`. @lajones any idea whether this field is actually needed for some reason?